build/cmake/lib/stc/CMakeLists.txt

```diff
@@ -164,31 +162,21 @@ target_include_directories(wxscintilla PRIVATE
     ${wxSOURCE_DIR}/src/stc/scintilla/lexlib
     ${wxSOURCE_DIR}/src/stc/scintilla/src
     )
-target_compile_definitions(wxscintilla PUBLIC
+target_compile_definitions(wxscintilla PRIVATE
     NO_CXX11_REGEX
     __WX__
     SCI_LEXER
     LINK_LEXERS
 )
 
-if(wxBUILD_PRECOMP)
-    # The auto-generated header causes undefined members and identifiers in the
-    # standard c++ headers when using clang.
-    # Do not disable precompiled headers entirely but use the main Scintilla
-    # header as prefix header so there is at least a small speedup.
-    if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
-        set(wxSCINTILLA_PREC_HEADER "${wxSOURCE_DIR}/src/stc/scintilla/include/Scintilla.h")
-    endif()
-    wx_target_enable_precomp(wxscintilla ${wxSCINTILLA_PREC_HEADER})
-endif()
-
+wx_target_enable_precomp(wxscintilla "${wxSOURCE_DIR}/src/stc/scintilla/include/Scintilla.h")
```

Comment:
Why remove the condition?

Reply:
The check for the condition is inside `wx_target_enable_precomp`.